tracer/src/Datadog.Trace/Debugger/LiveDebuggerFactory.cs

```diff
@@ -49,9 +47,18 @@ public static LiveDebugger Create(IDiscoveryService discoveryService, IRcmSubscr
             () => new MinimalAgentHeaderHelper(),
             uri => uri);
 
-        var batchApi = AgentBatchUploadApi.Create(apiFactory, discoveryService, gitMetadataTagsProvider);
-        var batchUploader = BatchUploader.Create(batchApi);
-        var debuggerSink = DebuggerSink.Create(snapshotStatusSink, probeStatusSink, batchUploader, debuggerSettings);
+        var snapshotBatchUploadApi = AgentBatchUploadApi.Create(apiFactory, discoveryService, gitMetadataTagsProvider, false);
```

Comment:
nit:
```suggestion
        var snapshotBatchUploadApi = AgentBatchUploadApi.Create(apiFactory, discoveryService, gitMetadataTagsProvider, isDiagnostics: false);
```

Reply:
Opened a PR for this https://github.com/DataDog/dd-trace-dotnet/pull/5396